src/index.html

```diff
@@ -1,12 +1,363 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
```

Comment:
```suggestion
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
```
I dont see usage of Montserrat font